mindsdb/integrations/handlers/ckan_handler/README.md

```diff
@@ -1,29 +1,78 @@
 ## CKAN Integration handler
 
-This handler is used to integrate with [CKAN](https://ckan.org/).
-CKAN is Data Catalogue for Open Data, and it stores data in a [DataStore](http://docs.ckan.org/en/2.9/maintaining/datastore.html). 
-To retrieve data from CKAN, you need to use the CKAN [API](https://ckan.org/docs/api/). 
+This handler facilitates integration with [CKAN](https://ckan.org/).
+an open-source data catalog platform for managing and publishing open data. CKAN organizes datasets and stores data in its [DataStore](http://docs.ckan.org/en/2.11/maintaining/datastore.html).To retrieve data from CKAN, the [CKAAPI](https://github.com/ckan/ckanapi) must be used.
 
+## Installation
 
-## Creating a CKAN API client
-Connecting to CKAN is done by creating a CKAN API client
-In this handler, you can create a CKAN API client with [ckanapi](https://github.com/ckan/ckanapi).
+To use the CKAN handler, you need to have MindsDB installed. If you haven't installed MindsDB yet, you can do so using pip:
 
-*Note: Some CKAN instances will require you to provide API key. You can find it in the CKAN user panel.
+```bash
+pip install mindsdb
```

Comment:
Hey @tino097,
We don't recommend installing MindsDB via PyPI anymore; can you please update this by instructing users to install it via Docker or Docker Desktop, please? [Here](https://github.com/mindsdb/mindsdb/tree/main/mindsdb/integrations/handlers/hana_handler) is an example.